app/src/main/java/com/whereuat/whereu/MainActivity.java

```diff
@@ -96,6 +109,23 @@ public void onReceive(Context context, Intent intent) {
             Toast.makeText(context, text, Toast.LENGTH_SHORT).show();
         }
     }
+
+    public void addContact(View view){
```

Comment:
Remove empty line at top of function.

Also, not for this PR but it might be nice to find a way to unify our permission requests at some point.